### PR TITLE
fix(openchallenges): make homepage more mobile-friendly

### LINKED ISSUE #2378: [Story] Optimize the OC home page for mobile
### What product(s) is this story for?

OpenChallenges

### As a user, I want

As a New User, I want OC home page to present well on my mobile device, so that I can focus on the page content.

### Description

The home page is one of the two most used landing pages along with the challenge search page. Also, most new users are on a mobile when they visit OC. for the first time.

The goal of this ticket is to start with optimizing the OC home page for mobile devices. The page currently looks OK but the layout and styles could be improved in terms of responsiveness.

Cc: @vpchung 

### Acceptance criteria

- All the sections of the home page have been reviewed
- The ticket includes a list of changes that we could implement to make the home page more responsive
- The changes are implemented in one or multiple PRs.

Out of scope:
- Pages that are not the home page

### Tasks

_No response_

### Anything else?

_No response_

### Have you linked this story to a GitHub Project?

- [X] I have linked this story to a GitHub Project and set its metadata.

### PR DESCRIPTION
**Pull request by vpchung**
Fixes #2378 

Note: this PR contains a subset of changes proposed in #2402 (now closed)

## Changelog

* adjust logo size of ITCR image when screen is smaller - this is what caused the black margin in the bug report
* adjust text size and label rotation in timeline plot

## TODO

Once orientation of plot is determined (see screenshots below), remove the unneeded code

## Preview

* **ITCR logo**
290030938-0c8c8f42-8eac-417c-9e54-9a1d1e164d3c

* **Timeline plot**
Screenshot 2024-02-01 at 4 35 01 PM

Screenshot 2024-02-01 at 4 34 42 PM

Alternatively, we can flip the plot so that the y-axis is the category (year) and the x-axis is the data:

Screenshot 2024-02-01 at 4 24 08 PM

Screenshot 2024-02-01 at 4 25 31 PM

### CODE REVIEW

File: libs/openchallenges/home/src/lib/statistics-viewer/statistics-viewer.component.ts [hunk 1]
```diff
@@ -37,24 +38,26 @@ export class StatisticsViewerComponent implements OnInit, OnDestroy {
...
                 type: 'value',
                 name: '',
-                axisLabel: { fontSize: '1em' },
                 nameTextStyle: {
-                  fontSize: '1.1em',
                   lineHeight: 56,
                 },
               },
             ],
+            grid: {
```
Comment (rrchai) [inline libs/openchallenges/home/src/lib/statistics-viewer/statistics-viewer.component.ts:58]: nice!

**Comment by rrchai** [comment]
The pr looks good! Thanks.

For the timeline plot, both horizontal and vertical work for me. If we do vertically, I prefer years are arranged in descending order.

**Comment by tschaffter** [comment]
I prefer vertical bars as the ""number of challenges goes up" effect is more intuitive going from left to right.

**Comment by vpchung** [comment]
Will move forward with the vertical bar chart:

Screenshot 2024-02-14 at 5 53 07 PM

**Review by tschaffter — APPROVED** [review]

**Pull request merged**